packages/plugin-jsdoc-tag-value/src/index.js

```diff
@@ -0,0 +1,53 @@
+/* @flow */
+
+const documentation = require('documentation');
+
+/**
+ * Render a jsdoc tag's value for a given file/function
+ * @example
+ * <!--emdaer-p
+ *   - '@emdaer/plugin-jsdoc-tag-value'
+ *   - source: ./src/index.js
+ *     functionName: jsdocTagValue
+ *     tag: example
+ * -->
+ * @param                     options
+ * @param   {string}          [options.source] The file that contains the function
+ * @param   {string}          [options.functionName] The function name that contains the desired jsdoc tag
+ * @param   {string}          [options.tag] The tag from whom to pull the value
```

Comment:
we may want to consider a tagIndex param. that way we could get the 2nd example, for example. default would be return all if there are multiple?

Reply:
☝️ this is done